census/tftransformestimator/trainer/task.py

```diff
@@ -0,0 +1,191 @@
+import argparse
+import os
+
+#import trainer.model as model
+import model
+
+import tensorflow as tf
+from tensorflow.contrib.training.python.training import hparam
+
+
+def run_experiment(hparams):
+  """Run the training and evaluate using the high level API"""
+  print("run")
```

Comment:
Do we want this here? Maybe we could provide more some useful information, or remove it.

Reply:
Done